src/vsr/replica.zig

```diff
@@ -4726,42 +4726,6 @@ pub fn ReplicaType(
             return message.ref();
         }
 
-        /// Discards uncommitted ops during a view change from after and including `op`.
-        /// This is required to maximize availability in the presence of storage faults.
-        /// Refer to the CTRL protocol from Protocol-Aware Recovery for Consensus-Based Storage.
-        fn primary_discard_uncommitted_ops_from(self: *Self, op: u64, checksum: u128) void {
```

Comment:
wait wat, wasn't this supposed to be used somewhere? 

Ah, ok, this was (not) removed in f74eda450b0a0fa687395e1a5cd97810af0b6345, `git log -S` is everuseful!

Reply:
Ah, TIL about `git log -S`, I'm going to use this one in the future! :D